tests/download-util.test.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Tests download-util.
+ */
+// import 'mocha';
+import { expect } from 'chai';
+
+import * as fs from 'fs';
+import * as os from 'os';
+import * as io from '@actions/io';
+
+import { TestToolCache, TEST_SDK_VERSION } from '../src/test-util';
+const [toolDir, tempDir] = TestToolCache.override();
+
+// Import modules being tested after test setup as run.
+import * as downloadUtil from '../src/download-util';
+
+import { getReleaseURL } from '../src/format-url';
+
+describe('#downloadAndExtractTool', function() {
+  beforeEach(async function() {
+    await io.rmRF(toolDir);
+    await io.rmRF(tempDir);
+  });
+
+  afterEach(async function() {
+    try {
+      await io.rmRF(toolDir);
+      await io.rmRF(tempDir);
+    } catch (err) {
+      console.error('Error occurred during test cleanup: ' + err);
+    }
+  });
+
+  it('downloads and extracts linux version', async function() {
+    if (os.platform() === 'win32') {
+      // https://github.com/actions/toolkit/issues/194
```

Comment:
this seems closed, might be worth running a test without this guard to confirm
https://github.com/actions/toolkit/issues/194